src/embed.ts

```diff
@@ -108,6 +108,11 @@ export abstract class Embed {
   config: IInternalEmbedConfiguration;
 
   /**
+   * Url used in the load request.
+   */
+  loadPath: string;
```

Comment:
Typescript doesn't seem to support abstract properties.  I think technically it would be better to use abstract method such as `abstract getLoadPath(): string` which allows proper enforcement that the method is implemented in concrete classes but I thought the semantics of it being a method which returned hardcoded string were worse.